Builder/src/test/java/devslp/designpatterns/builder/BuilderTest.java

```diff
@@ -0,0 +1,95 @@
+package devslp.designpatterns.builder;
+
+import devslp.designpatterns.builder.Builders.HappyMealBuilder;
+import devslp.designpatterns.builder.Builders.IndividualComboBuilder;
+
+import static org.junit.Assert.assertEquals;
+
+import devslp.designpatterns.builder.Items.Burger;
+import devslp.designpatterns.builder.Items.Drink;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by mackaber on 27/03/17.
+ */
+public class BuilderTest {
+
+    @Test
+    public void individualComboPrice() {
+        Employee employee = new Employee();
+        employee.setMealBuilder(new IndividualComboBuilder());
+        employee.makeMeal();
+
+        Meal meal = employee.getMeal();
+        assertEquals(meal.getTotal(), 50.0, 0);
+    }
+
+    @Test
+    public void happyMealPrice() {
+        Employee employee = new Employee();
+        employee.setMealBuilder(new HappyMealBuilder());
+        employee.makeMeal();
+
+        Meal meal = employee.getMeal();
+        assertEquals(meal.getTotal(), 63.0, 0);
+    }
+
+    @Test
+    public void ItemCount() {
+        Employee employee = new Employee();
+        List<Meal> meals = new ArrayList<>();
```

Comment:
In this case the use of the List meals does not add valuable instructions to test the case and neither to make it clearer. Instead, it adds unnecessary complexity. 

As far as I can see, What you want to test is number of items in the meal after call the method makeMeal, so you can only add 

assertEquals("Evaluation fails", meal.getItems().size(), 3); 

after makeMeal call instead of adding the meal to the list